packages/fast-components-react-msft/package.json

```diff
@@ -73,6 +73,7 @@
     "@microsoft/fast-glyphs-msft": "^2.2.0",
     "@microsoft/fast-jest-snapshots-react": "^2.2.0",
     "@microsoft/fast-jss-manager": "^2.2.0",
+    "@microsoft/fast-jss-utilities": "^2.2.0",
```

Comment:
This should be removed and the styles should be exported from the styles package.